libCacheSim/bin/cachesim/sim.c

```diff
@@ -27,6 +27,10 @@ void simulate(reader_t *reader, cache_t *cache, int report_interval,
 
   double start_time = -1;
   while (req->valid) {
+    if (req_cnt < 10) {
```

Comment:
Can we change this to a parameter and print only two requests? The main concern is that the output will be too large when running millions of simulations. We can print two at INFO and a few at DEBUG.